ci/terraform/shared/kms.tf

```diff
@@ -449,4 +449,172 @@ resource "aws_kms_key" "authentication_callback_userinfo_encryption_key" {
 resource "aws_kms_alias" "authentication_callback_userinfo_encryption_key_alias" {
   name          = "alias/${var.environment}-authentication-callback-userinfo-encryption-key-alias"
   target_key_id = aws_kms_key.authentication_callback_userinfo_encryption_key.key_id
+}
```

Comment:
Looks like there's an issue with the lambdas not being able to decrypt.  Need to make some addtions.